libs/librrgraph/src/base/rr_graph_builder.h

```diff
@@ -351,6 +351,17 @@ class RRGraphBuilder {
     inline void alloc_and_load_edges(const t_rr_edge_info_set* rr_edges_to_create) {
         node_storage_.alloc_and_load_edges(rr_edges_to_create);
     }
+    
+    /** @brief Removes a given list of RREdgeIds for the RR Graph.
+     * This method does not preserve the order of edges. If you're
+     * calling it after partition_edges has been called, you need
+     * to call it again.
+     *
+     * @param rr_edges_to_remove list of RREdgeIds to be removed
+     */
+    inline void remove_edges(std::vector<RREdgeId>& rr_edges_to_remove) {
+        node_storage_.remove_edges(rr_edges_to_remove);
```

Comment:
I think we set a flag after partitioning edges. I'd suggest adding an assertion to ensure that the flag is not set when this function is called.

Reply:
I think the routine is legal to call after partition_edges; it should just unset the edges are partitioned flag.